src/app/entities/categories.ts

```diff
@@ -0,0 +1,10 @@
+import { ObjectId } from 'mongoose';
+import { Guides } from './guides';
+
+export interface Categories {
+  _id: ObjectId;
+  _idGuides: ObjectId;
+  _idParent?: ObjectId;
```

Comment:
Tendo em vista que guias(Guides) podem ter ou não categorias(Categories), e categorias podem ter ou não outras categorias uma sugestão seria:

```suggestion
export interface Categories {
  _id: ObjectId;
  title: string;
  content: string;
  categories?: Categories[];
```

Ficaria algo assim no MongoDB:
    _id :6169e2f2eb479cbd8bf7da42
    title :"Acessibilidade motora"
    content : "Necessidade de veículo para locomoção"
    categories  :  Array
        0 :6169d7bf6b069e1d75df79f2
        1 :6169d7c06b069e1d75df79f4
    __v
    :0

Reply:
Isso não se aplica a entiade de `Categoria`, apenas a de `Guides`.